opennms-correlation/drools-correlation-engine/src/test/opennms-home/etc/drools-engine.d/droolsFusion/DroolsFusion.drl

```diff
@@ -34,6 +36,22 @@ then
         engine.sendEvent(eventBuilder.getEvent());
 end
 
+
+rule "this-rule-should-throw-exception"
+       salience 400
+when    $e : Event(uei matches "uei.opennms.org/triggerTestForFusion")
+
+then
+         System.err.println("running exception rule, this will throw exception");
+         EventBuilder eventBuilder = new EventBuilder("uei.opennms.org/triggerTestForFusion", "Component Correlator");
+         // Code to generate ConcurrentModification Exception. see NMS-10363

```

Comment:
I guess we could also just directly throw the CME instead of trying to replicate it.